### PR TITLE
Animated raptorborgs update

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
Animates all raptor borg sprites, and also separates them into the large sprites section for ease of modification. This way, we won't have to change every other sprite to be as big as it.

## Why It's Good For The Game
They Walnk

## Proof Of Testing
(Note this was recorded pre-pixel fix and the jank is gone now)
https://github.com/Bubberstation/Bubberstation/assets/110273561/0e14fc0a-6d47-42ad-bc1c-829b16eb5609

## Changelog

:cl: ReturnToZender, SpaceManJim (sprites)
add: Animated raptorborg sprites
/:cl:

### CODE REVIEW

File: _maps/skyrat/automapper/automapper_config.toml [hunk 1]
```diff
@@ -22,8 +22,8 @@ trait_name = "CentCom"
 [templates.centcom_shuttle_dock]
 map_files = ["centcom_shuttle_dock.dmm"]
 directory = "_maps/skyrat/automapper/templates/centcom/"
-required_map = "builtin"
-coordinates = [206, 85, 1]
+required_map = "CentCom_skyrat_z2.dmm" ## BUBBERSTATION EDIT - MOVES THE SHUTTLE TO THE CENTCOM DOCK
+coordinates = [51, 167, 2]			   ## BUBBERSTATION EDIT
```
Comment (Cyprex) [inline _maps/skyrat/automapper/automapper_config.toml:26]: Atomize this into another PR

Comment (ReturnToZender) [reply _maps/skyrat/automapper/automapper_config.toml]: I didn't do this part, it was just pulling master

File: html/changelogs/AutoChangeLog-pr-964.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+author: "thegrb93"
+delete-after: True
+changes:
+  - rscadd: "Borg dominatrix module to lustwish that gives them some dominatrix toys"
```
Comment (Cyprex) [inline html/changelogs/AutoChangeLog-pr-964.yml]: No idea if these changelog entries shoudl be here

File: modular_zubbers/modules/borgs/code/robot_model.dm [hunk 3]
```diff
@@ -29,8 +29,8 @@
 	)
 	special_light_key = null
 	borg_skins = list(
-		"Vale" = list(SKIN_ICON_STATE = "valecc", SKIN_ICON = CYBORG_ICON_CENTCOM_WIDE_BUBBER, SKIN_FEATURES = list(TRAIT_R_UNIQUEWRECK, TRAIT_R_WIDE)),
-		"Raptor" = list(SKIN_ICON_STATE = CYBORG_ICON_TYPE_RAPTOR, SKIN_ICON = CYBORG_ICON_CENTCOM_WIDE_BUBBER, SKIN_FEATURES = list(TRAIT_R_UNIQUEWRECK, TRAIT_R_WIDE)), //aKhroma :)
+		"Raptor" = list(SKIN_ICON_STATE = CYBORG_ICON_TYPE_RAPTOR, SKIN_ICON = CYBORG_ICON_CENTCOM_LARGE_BUBBER, SKIN_FEATURES = list(TRAIT_R_UNIQUEWRECK, TRAIT_R_LARGE), SKIN_PIXEL_X = -(16)), //aKhroma :)
```
Comment (Cyprex) [inline modular_zubbers/modules/borgs/code/robot_model.dm:32]: This line looks busted as heck

File: modular_zubbers/modules/borgs/code/robot_model.dm [hunk 4]
```diff
@@ -48,61 +48,115 @@
...
+			SKIN_ICON = CYBORG_ICON_SYNDIE_LARGE_BUBBER,
+			SKIN_FEATURES = list(TRAIT_R_UNIQUEWRECK, TRAIT_R_LARGE),
+			SKIN_PIXEL_X = -(16)
+		),
+		"Saboraptor" = list(
+			SKIN_ICON_STATE = "saboraptor",
+			SKIN_ICON = CYBORG_ICON_SYNDIE_LARGE_BUBBER,
+			SKIN_FEATURES = list(TRAIT_R_UNIQUEWRECK, TRAIT_R_LARGE),
+			SKIN_PIXEL_X = -(16)
+		),
```
Comment (Cyprex) [inline modular_zubbers/modules/borgs/code/robot_model.dm:113]: All this code duplication is crap, each skin should only need to be defined once

File: html/changelogs/AutoChangeLog-pr-964.yml (hunk 2 above)
Comment (ReturnToZender) [reply html/changelogs/AutoChangeLog-pr-964.yml]: No longer in

File: modular_zubbers/modules/borgs/code/robot_model.dm (hunk 3 above)
Comment (ReturnToZender) [reply modular_zubbers/modules/borgs/code/robot_model.dm]: Fixed now

File: modular_zubbers/modules/borgs/code/robot_model.dm (hunk 4 above)
Comment (ReturnToZender) [reply modular_zubbers/modules/borgs/code/robot_model.dm:129]: Unfortunately, there are multiple robot models

**Review by Majkl-J — APPROVED** [review]
Sprite updates

**Pull request merged**